pkg/kubectl/genericclioptions/resource/visitor_test.go

```diff
@@ -100,3 +104,202 @@ func TestVisitorHttpGet(t *testing.T) {
 	assert.NotNil(t, actualBytes)
 	assert.Equal(t, 2, i)
 }
+
+func TestVisitorRefresh(t *testing.T) {
+	testcases := []struct {
+		description   string
+		obj           runtime.Object
+		accessor      TestMetadataAccessor
+		ignoreError   bool
+		expectedError bool
+		expectedInfo  Info
+	}{
+		{
+			description: "error from Name()",
+			accessor: TestMetadataAccessor{
+				MockName: func(obj runtime.Object) (string, error) {
+					return "", fmt.Errorf("Error getting object's name")
```

Comment:
lower-case errors

Reply:
Thank you for the code review. I have updated the PR to change to lower case errors.